colossalai/inference/build.sh

```diff
@@ -9,7 +9,7 @@ mkdir 3rdParty
 cd 3rdParty
 git clone https://github.com/ModelTC/lightllm 
 cd lightllm
-git checkout 28c1267cfca536b7b4f28e921e03de735b003039
+git checkout ece7b43f8a6dfa74027adc77c2c176cff28c76c8
```

Comment:
As requirements are moved to requirements-infer.txt, is this file useless any more?